bin/ch/ch.cpp

```diff
@@ -583,7 +583,11 @@ int main(int argc, char** argv)
         Helpers::NarrowStringToWideDynamic(argv[i], &args[i]);
     }
 
-    int ret = wmain(argc, args);
+    // Call wmain with a copy of args, as HostConfigFlags may change argv
+    char16** argsCopy = new char16*[argc];
+    memcpy(argsCopy, args, sizeof(args[0]) * argc);
```

Comment:
This is an interesting use of `memcpy` to convert from `char` to `char16` but I'm not sure if it works?